src/store/store.tsx

```diff
@@ -0,0 +1,31 @@
+// src/app/store.js
+import { configureStore } from '@reduxjs/toolkit';
+import expenseFormReducer from '../modules/expense/expenseFormSlice';
+import dayjs, { Dayjs } from 'dayjs';
+
+interface RootState {
+  expenseForm: {
+    category: string[];
+    textValue: string;
+    selectedWallet: string;
+    showAttachmentComponent: boolean;
```

Comment:
This should not be in the redux, it's a local state that should be in the component itself